lib/looker-sdk/client/dynamic.rb

```diff
@@ -53,20 +53,17 @@ def load_swagger
         @swagger ||= without_authentication { try_load_swagger }
 
         unless @swagger
-          # capture the bits we may need later, avoiding potential buffer reuse in last_response between requests
-          response_wo_auth_status = last_response&.status
-          response_wo_auth_data = last_response&.data
-
           # try again, this time with authentication
           @swagger = try_load_swagger
         end
 
         # in unit tests, @swagger may be nil and last_response nil because no HTTP request was made
-        if @swagger.nil? && (response_wo_auth_status || last_response)
-          msg = "Load of swagger.json failed. "
-          msg << "Without authentication HTTP response status: (#{response_wo_auth_status}) data: #{response_wo_auth_data}. " if response_wo_auth_status
-          msg << "WITH authentication HTTP response status: (#{last_response.status}) data: #{last_response.data}" if last_response
-          looker_warn(msg)
+        if @swagger.nil?
+          if @last_error
+            raise @last_error
+          else
+            raise "Load of swagger.json failed."
+          end
```

Comment:
no swagger then no need to continue execution. this should stomp out most of the `missing method` errors we get in the wild.